app/src/androidTest/java/com/github/onlynotesswent/ui/editNote/EditNoteTest.kt

```diff
@@ -40,7 +42,15 @@ class EditNoteTest {
 
   @Test
   fun displayBaseComponents() {
-    composeTestRule.onNodeWithTag("EditNote text").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("GoBack button").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("EditNote textField").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("EditTitle textField").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Save button").assertIsDisplayed()
+  }
+
+  @Test
+  fun saveClickCallsNavActions() {
+    composeTestRule.onNodeWithTag("Save button").assertIsDisplayed()
```

Comment:
The save button being displayed was already tested in the previous method, so there is no need to call it here